app/views/users/index.html.erb

```diff
@@ -1,2 +1,16 @@
-<h1> Here is list of users and their corresponding number of posts
-</h1>
+<div class="users-index-wrapper">
+<h1 class="header-text">Here is list of users and their corresponding number of posts</h1>
+
+<ul class="users-container">
+  <% @users.each do |user| %>
+    <li class="author-info user-link-container">
+      <img class="post-box" src ="/box-post.png" alt="Post logo">
```

Comment:
- [ ] This image has alt attribute which is a good practice. But it should render users' photo here. You can write the right alt attribute might be something like 'user-photo'. And iti should render the photo as the source.
```
<%= image_tag(user.photo) %>
```

Reply:
kindly explain further i don't get what you mean by that